templates/catalog/product.tpl

```diff
@@ -116,6 +117,7 @@
 
   {* SECOND PART - REASSURANCE, TABS *}
   <div class="row">
```

Comment:
Just need to add some space before row.

```suggestion
  <div class="row mt-5">
```

Reply:
Checked again, it's doesn't need any changes!
It works fine :100: 
Very nice :clap:

Reply:
I meant: instead of `mt-5` :)

Reply:
Try to to build and test this PR, there is no problem for spacing.
I got wrong because forget to clear cache!